microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -106,29 +102,81 @@ Now you can use Microsoft Edge DevTools to inspect and debug the WebView2 contro
+
+1.  Verify that you're redirected to the **Apps manager: Windows Device Portal** page:
+
+    ![Device Portal](./remote-debugging-images/device-portal.png)
+
+1.  Go to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page loads an empty list: `[]`, unless there's an application running that is debuggable by remote tools.
+
+1.  Configure your WebView2 WinUI 2 (UWP) app for remote debugging:
+
+    To enable remote debugging, two environment variables must be set in your project.  These variables must be set before creating the `CoreWebView2` instance, and before either setting the `WebView2.Source` property or calling the `WebView2.EnsureCoreWebView2Async` method.
+
+    Set the following variable:
 
-1. Install [Remote Tools for Microsoft Edge](https://www.microsoft.com/store/productId/9P6CMFV44ZLT) from the Microsoft Store on your Hololens 2 device.
+    ```
+    "WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging"
+    ```
+    
+    For example, if you're using the [WebView2 Sample UWP app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/webview2_sample_uwp), you can set the environment variable by adding the following line in the `Browser.xaml.cs` file:
+    
+    ```csharp
+    Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--enable-features=msEdgeDevToolsWdpRemoteDebugging");
+    ```
+
+    The line is numbered 33 in this screenshot, in the `Browser()` constructor, below an `#endif` that wraps an existing `SetEnvironmentVariable` statement:
 
-1. Follow steps 8-12 under [Debugging desktop WebView2 WinUI 2 (UWP) apps](#debugging-desktop-webview2-winui-2-uwp-apps).
+    ![Browser Arguments Remote Debugging](./remote-debugging-images/browser-arguments-remote-debugging.png)
 
-1. Launch your app on your Hololens 2 device.
+<!-- / steps 8-12 -->
 
-1. Follow steps 14 & 15 under [Debugging desktop WebView2 WinUI 2 (UWP) apps](#debugging-desktop-webview2-winui-2-uwp-apps).
+1.  Launch your app on your Hololens 2 device.
 
-1. Verify that you can successfully connect, so that your debuggable WebView2 control, named **Edge**, is listed under your machine name:
+<!-- steps 14-15: -->
+1.  Go to `http://<Device Portal URL>/msedge`, such as `http://localhost:50080/msedge`, and make sure your WebView2 instance appears in the list:
+
+    ![Debuggable WebView2 Instance](./remote-debugging-images/debuggable-webviews.png)
+
+1.  Go to `edge://inspect`.  In the **Connect to a remote Windows device** text box, enter `http://<Device Portal URL>`, such as `http://localhost:50080`, and then click **Connect to Device**.
+
+<!-- / steps 14-15 -->
+
+1.  Verify that you can successfully connect, so that your debuggable WebView2 control, named **Edge**, is listed under your machine name:

```

Comment:
maybe the left-justified hidden comments reset numbering, and I plan to move them to col 4 or 5, it would be good to see if that fixes.  @maurawinstanley